service/controller/clusterapi/cluster.go

```diff
@@ -42,12 +42,12 @@ type ClusterConfig struct {
 	DeleteLoggingBucket        bool
 	DockerDaemonCIDR           string
 	EncrypterBackend           string
-	GuestAWSConfig             ClusterConfigAWSConfig
+	GuestAvailabilityZones     []string
```

Comment:
Making the config simpler. This guest thing is not used at all and it was also abused to carry some AZ information.